site/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "private": true,
   "devDependencies": {
-    "gulp": "^3.8.8"
+    "gulp": "^3.8.8",
+    "gulp-util": "^3.0.7"
```

Comment:
As discussed with @adamwathan in https://github.com/tightenco/jigsaw/pull/23#issuecomment-200310852 `gulp-util` is now an explicit dependency declared in Jigsaws `package.json` in case Elixir's dependencies ever change.